src/gdm/resilience/distribution_resilience_system.py

```diff
@@ -0,0 +1,11 @@
+from infrasys import System
+
+import gdm
+
+
+class DistributionResilenceSystem(System):
```

Comment:
A few questions.
1. Are you sure you want to call this "DistributionResilienceSystem"?  That name links these components to a specific use case (resilience). I liked some of the suggestions that ChatGPT gave me for names:

![image](https://github.com/user-attachments/assets/e4db1672-de5b-49ea-91d8-f2e4066813e0)

3. Does CIM have the same level of detail for poles?  This might be another good way to distinguish GDM.  
4. How many of these attributes are used in fragility curves?